versions.csv

```diff
@@ -1,10 +1,10 @@
 AssemblyPath, Version
-iothub\device\src\Microsoft.Azure.Devices.Client.csproj, 1.36.6
-iothub\service\src\Microsoft.Azure.Devices.csproj, 1.31.3
-shared\src\Microsoft.Azure.Devices.Shared.csproj, 1.27.1
-provisioning\device\src\Microsoft.Azure.Devices.Provisioning.Client.csproj, 1.16.4
+iothub\device\src\Microsoft.Azure.Devices.Client.csproj, 1.36.7
+iothub\service\src\Microsoft.Azure.Devices.csproj, 1.31.4
+shared\src\Microsoft.Azure.Devices.Shared.csproj, 1.27.2
+provisioning\device\src\Microsoft.Azure.Devices.Provisioning.Client.csproj, 1.16.5
```

Comment:
This csproj update seems to be missing.

Reply:
If this project isn't required to be updated, then you'll need to remove this entry from the .csv file